app/assets/src/components/common/ProjectCreationForm.jsx

```diff
@@ -96,7 +96,9 @@ class ProjectCreationForm extends React.Component {
           <div
             className={cx(
               cs.createButton,
-              this.state.name === "" && cs.disabled
+              this.state.name === "" &&
+                this.state.publicAccess === -1 &&
+                cs.disabled
             )}
             onClick={this.handleCreateProject}
```

Comment:
You'll also need to check `publicAccess` in `handleCreateProject`.

We should eventually turn this into a different style of "button", so you can just set disabled once, and it'll handle both the styling and the behavior.

Reply:
👍